Install/Program Files to Install/bundle/Contents/Resources/ADSKIFCExporterHelp.htm

```diff
@@ -238,6 +238,32 @@ <h2 id="VersionHistory" name="VersionHistory">Version History</h2>
 
 <div class="versionset">
 
+<div class="version">22.4.0.0</div><div class="versioninfo-bullet">
+
+      <br><p>
+         General:
+			<ul>
+            <li>This version contains various improvements and bugfixes since the latest release of 22.3.0.0.
+			</ul>	
+			<br>
+		</p>
+     <p>
+	  		Improvements:
+			<ul>
+				<li>Updated Autodesk.SteelConnections.ASIFC.dll.
+				<li>Updated code for compatibility with Revit 2022.1.1 API.
+			</ul>
+	  </p>
+	  <br> <p>
+         Bug Fixes:
+	  </p>
+			<ul>
+				<li>Fixed bug with steel connections elements exporting.
+				<li>Fixed number of bugs related to differences in previous Revit API versions.

```

Comment:
Are these the only 22.4 changes?

Reply:
Yes, it's mostly fixing compatibility with different versions of Revit.